js/iframe-coordinator/libs/index.ts

```diff
@@ -1,5 +1,4 @@
-import client from './client';
```

Comment:
I don't think this is even used in our module now.  I personally still think it would be better to just export index and let them consume It like `import {Client} from 'iframe-coordinator`

Reply:
I agree that that's a nicer API for importing. My only question is would that result in the host & client code getting bundled with every application, or the latest approaches to tree-shaking able to recognize that the host code in `index` is never used and exclude it from the build? I didn't think they were capable of that last time I looked, but I haven't done a deep dive into it.